docs/operations/data_generator.md

```diff
@@ -0,0 +1,91 @@
+
```

Comment:
The filename here should be same the processor, `elasticsearch_data_generator `.  Also there some whitespace at the top of file that should be removed.